relay/server.go

```diff
@@ -78,14 +78,14 @@ type Config struct {
 	// goroutines.
 	MetadataMaxConcurrency int
 
-	// BlobCacheSize is the maximum number of items in the blob cache.
-	BlobCacheSize int
+	// BlobCacheSize is the maximum size of the blob cache, in bytes.
+	BlobCacheSize uint64
```

Comment:
May call it `BlobCacheSizeBytes` and then it's self-explaining

Reply:
done